environments/sandbox/ss-service-core-compute-sandbox-internal.yml

```diff
@@ -28,4 +28,8 @@ A:
     record:
     - 10.140.15.250
     ttl: 300
+  - name: docmosis-aks
```

Comment:
do you need to suffix it with -aks? it appears to be in a different zone to docmosis normally?

Reply:
The suffix isn't required, I just wanted to make a clear distinction between the Docmosis services running on the virtual machines and those running in AKS.  Happy to change it.

Reply:
....This was only for testing